deploy/helm/scf/templates/eirini.yaml

```diff
@@ -0,0 +1,63 @@
+# A service for the cc uploader
```

Comment:
This file should be rendered only when Eirini feature is enabled.

Reply:
👍